src/ensembl/src/shared/components/accordion/components/Accordion.test.tsx

```diff
@@ -152,18 +180,29 @@ describe('Accordion', () => {
           </Accordion>
         );
 
-        wrapper.find(AccordionItemButton).simulate('click');
-        wrapper.find(AccordionItemButton).simulate('click');
+        userEvent.click(
+          container.querySelector(
+            '.accordionButtonDefault'
+          ) as HTMLButtonElement
+        );
+        userEvent.click(
+          container.querySelector(
+            '.accordionButtonDefault'
+          ) as HTMLButtonElement
+        );
```

Comment:
It's clicking the same element two times, isn't it? And is also reusing this element below? Wouldn't the intention be clearer if the result of the `querySelector` method were assigned to a variable?

Reply:
That's done